### PR TITLE
cifs-utils: unbreak musl

### PR DESCRIPTION
**Pull request by pullmoll**

### CODE REVIEW

File: srcpkgs/cifs-utils/template [hunk 1]
```diff
@@ -14,6 +14,15 @@ homepage="http://wiki.samba.org/index.php/LinuxCIFS_utils"
 distfiles="https://ftp.samba.org/pub/linux-cifs/$pkgname/$pkgname-$version.tar.bz2"
 checksum=38fc63926af435dae4ebcf4406275580a692d9fb9ee3e32170317cf2ba68e6e3
 
+pre_configure() {
+	# Fix musl build
+	sed -e "/#include <stdlib.h>/a #include <paths.h>\n#include <limits.h>" \
+		-i ${wrksrc}/mtab.c
+	sed -e "/#include <stdlib.h>/a #include <linux/limits.h>\n#include <endian.h>" \
```
Comment (beefcurtains) [inline srcpkgs/cifs-utils/template]: not ``sys/limits.h``?

Comment (pullmoll) [inline srcpkgs/cifs-utils/template]: I grepped for XATTR_SIZE_MAX in the masterdir and it's defined only in linux/limits.h

Comment (beefcurtains) [inline srcpkgs/cifs-utils/template]: Alrighty

**Comment by beefcurtains** [comment]
:+1: but please try to limit the number of columns to ~80, thanks :)

**Comment by pullmoll** [comment]
Should I break the sed /a to multiple lines then? I must admin that my terminal windows have 112 columns :)

**Comment by beefcurtains** [comment]
There's nothing in ``CONTRIBUTING.md`` about this, so it's probably ok as-is.

**Comment by pullmoll** [comment]
There you go

**Comment by beefcurtains** [comment]
That looks really nice, thank you very much kind sir!

**Comment by pullmoll** [comment]
*lol* :fork_and_knife: :beer: anything else?

**Pull request merged**